### PR TITLE
Hugo site needed docker file added for BND

### PR DESCRIPTION
**Pull request by Darshan072**
- Added docker file that will help us to deploy the hugo site
- Updated config file for base url
- Added ngnix for static site deployment

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseURL = 'https://anthoneous.github.io'
+baseURL = 'https://anthoneous-darshan-darshan072-main.pacenthink.co/'
```
Comment (anthoneous) [inline config.toml:1]: I believe this needs to be changed but I will merge and test as is.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.17.1 AS builder
+ARG VERSION=0.110.0
+ADD https://github.com/gohugoio/hugo/releases/download/v0.110.0/hugo_extended_0.110.0_darwin-universal.tar.gz /hugo.tar.gz
+RUN tar -zxvf hugo.tar.gz && \
+    rm -rf hugo.tar.gz && \
+    apk add --no-cache git
```
Comment (anthoneous) [inline Dockerfile:6]: What purpose is this serving?

**Review by anthoneous — APPROVED** [review]

File: config.toml (hunk 1 above)
Comment (Darshan072) [reply config.toml:1]: Yes this is needs to be changes based on the base url you will get from BND once first time deployment is done. or we can keep it empty (Blank) as well.

File: Dockerfile (hunk 2 above)
Comment (Darshan072) [reply Dockerfile:6]: On this we are pulling the HUGO environment image to our docker container so our site can run. That file we downloaded that is zip file so we are just doing extracting and removing the zip and git as well.

**Pull request merged**